database/docker-compose.yml

```diff
@@ -6,16 +6,14 @@ services:
       POSTGRES_DB: waect
       POSTGRES_USER: user
       POSTGRES_PASSWORD: pass
-      DB_HOST: host.docker.internals
```

Comment:
If you delete this it still works in your mac?

Reply:
I dont know. I added it again. Will check in the PR for the API and Docker stuff